### PR TITLE
make test coverage to 100%

### PR DESCRIPTION
**Pull request by sendilkumarn**

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ describe('utils', () => {
 			expect(utils.createAssetFilterFunction('js')).toMatchSnapshot();
 		});
 	});
+	describe('parseValue', () => {
+		it('should parse value an require statement', () => {
+			expect(utils.parseValue('test;test')).toMatchSnapshot();
```
Comment (ev1stensberg) [inline __tests__/index.test.js]: Would you mind testing this against an regexp like `\t` instead?

File: __tests__/index.test.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ describe('utils', () => {
 	});
 	describe('parseValue', () => {
 		it('should parse value an require statement', () => {
-			expect(utils.parseValue('test;test')).toMatchSnapshot();
+			expect(utils.parseValue('\t')).toMatchSnapshot();
```
Comment (ev1stensberg) [inline __tests__/index.test.js]: Awesome! Can you look at the raw value of the value instead? It's how it's meant to be used

Comment (sendilkumarn) [reply __tests__/index.test.js]: Thanks 👍  
Sorry, I didn't get what you are saying here?

Comment (ev1stensberg) [reply __tests__/index.test.js]: We're checking a regexp ( from https://github.com/webpack/webpack-cli/blob/master/lib/utils/ast-utils.js#L374 ) . Would be nice to know its rawValue and compare that. Could you find that in the result of `utils.parseValue` some `console.logs` and you'll find it!

**Comment by ev1stensberg** [comment]
Okay sorry for delaying. Having a look again tomorrow!

**Comment by ev1stensberg** [comment]
Think jscodeshift changed their internals a bit, so this is okay 👍

**Pull request merged**